### PR TITLE
Add a Divider block

### PR DESCRIPTION
**Pull request by praem90**
Divider block for the Issue #4

### CODE REVIEW

**Comment by zusamarehan** [comment]
Hi @praem90 

Thanks for taking this up.
Please add screenshots, and update the `README.md` file

File: src/Blocks/Divider.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace RehanKanak\LaravelNotionRenderer\Blocks;
+
+class Divider extends Block
+{
+
+    public function process(): Divider
+    {
+		$this->result .= '<div class="divide-y"></div>';
```
Comment (zusamarehan) [inline src/Blocks/Divider.php:10]: Please run Laravel Pint to avoid styling issues.

**Review by zusamarehan — CHANGES_REQUESTED** [review]

**Comment by praem90** [comment]
Attaching the screenshot using 
screenshot-notion test-2022 10 06-00_00_56

**Comment by zusamarehan** [comment]
Thanks for helping out @praem90

**Pull request merged**